app/models/register/EntityDetails.scala

```diff
@@ -37,11 +37,11 @@ sealed trait EntityDetails {
   case class EstablisherIndividualName(name: String) extends EntityDetails {
     override def route(id: Int, id2: Option[Int]): (String, String) =
       name ->
-        controllers.register.establishers.individual.routes.EstablisherDetailsController.onPageLoad(CheckMode, id).url
+        controllers.register.establishers.individual.routes.EstablisherDetailsController.onPageLoad(NormalMode, id).url
```

Comment:
Why would these be in NormalMode? These are used for the edit links on the AddEstablisher pages